components/automate-cli/pkg/verifyserver/constants/externalpostgresqlconstants.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+const (
+	EXTERNALPGSUCCESSCONNECTIONTITLE  = "Machine is able to connect with External Managed Postgres"
+	EXTERNALPGFAILCONNECTIONTITLE     = "External Postgresql Connection failed"
+	EXTERNALPGCONNECTIONERRORMSG      = "Machine is unable to connect with External Managed Postgresql"
```

Comment:
it should be EXTERNAL_PG_CONNECTION_ERROR_MSG = "Machine is unable to connect with external managed PostgreSQL"

Reply:
done